pkg/arvo/lib/bitcoin.hoon

```diff
@@ -57,7 +57,9 @@
     ~/  %to-cord
     |=  a=address  ^-  cord
     ?:  ?=([%base58 *] a)
-      (scot %uc +.a)
+      %-  crip
+      %+  slag  2
+      (trip (scot %uc +.a))
```

Comment:
> If there's a much faster way to do this operation just by manipulating the `@ta` directly, fine to switch to that.

```suggestion
      (rsh 3^2 (scot %uc +.a))
```

(Also for future reference, you might use `+scow` in place of `(trip (scot ...))`.)